lib/galaxy/config/schemas/config_schema.yml

```diff
@@ -3609,3 +3609,11 @@ mapping:
         default: true
         required: false
         desc: Display built-in converters in the tool panel.
+
+      enable_beacon_integration:
+        type: bool
+        default: false
+        required: true
```

Comment:
Is this really required?

Reply:
Is the question if the admin setting in general is needed? If so, then yes. It's a setting that potentially will expose data ... So better to have a triple op-in ;-)

Reply:
I was referring to the `required: true` in the schema. I understand this is an optional config option, if this is what that required attribute in the schema means :laughing:

Reply:
that is very true :)

Reply:
I set it to `required: false` now :+1: